### PR TITLE
Renames reports_wsgi.ini

### PR DESCRIPTION
**Pull request by souravsingh**
This Pull Request renames reports_wsgi.ini to reports.ini

### CODE REVIEW

**Comment by martenson** [comment]
Hi @souravsingh, at least `run_reports.sh`, `scripts/sync_reports_config.py`, and `.gitignore` should be also changed to reflect this.

**Comment by souravsingh** [comment]
@martenson Thanks for pointing it out.I will make those changes.

**Comment by souravsingh** [comment]
Does the PR look good?

File: run_reports.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # Description: This script can be used to start or stop the galaxy
 # reports web application. Passing in --sync-config as the first
 # argument to this will cause Galaxy's database and path parameters
-# from universe_wsgi.ini to be copied over into reports_wsgi.ini.
+# from universe_wsgi.ini to be copied over into reports.ini.
```
Comment (martenson) [inline run_reports.sh]: universe_wsgi.ini does not exist anymore, should ge `galaxy.ini`

File: .gitignore [hunk 2]
```diff
@@ -42,7 +42,7 @@ reports_webapp.pid
 
 # Config files
 universe_wsgi.ini
-reports_wsgi.ini
+reports.ini
```
Comment (nsoranzo) [inline .gitignore:46]: I guess you should just add `reports.ini` without removing `reports_wsgi.ini`.

Comment (souravsingh) [inline .gitignore:46]: Do we have to replace `universe_wsgi.ini` everywhere since it is replaced by `galaxy.ini`,as mentioned by @martenson?

Comment (nsoranzo) [inline .gitignore:46]: This has already be done (almost) everywhere.

File: run_reports.sh [hunk 3]
```diff
@@ -21,12 +21,12 @@ fi
...
-        GALAXY_REPORTS_CONFIG=config/reports_wsgi.ini
+    if [ -f reports.ini ]; then
+        GALAXY_REPORTS_CONFIG=reports.ini
+    elif [ -f config/reports.ini ]; then
+        GALAXY_REPORTS_CONFIG=config/reports.ini
     else
-        GALAXY_REPORTS_CONFIG=config/reports_wsgi.ini.sample
+        GALAXY_REPORTS_CONFIG=config/reports.ini.sample
     export GALAXY_REPORTS_CONFIG
     fi
```
Comment (nsoranzo) [inline run_reports.sh:32]: This is wrong, should be:

```
if [ -z "$GALAXY_REPORTS_CONFIG" ]; then
    if [ -f reports_wsgi.ini ]; then
        GALAXY_REPORTS_CONFIG=reports_wsgi.ini
    elif [ -f config/reports_wsgi.ini ]; then
        GALAXY_REPORTS_CONFIG=config/reports_wsgi.ini
    elif [ -f config/reports.ini ]; then
        GALAXY_REPORTS_CONFIG=config/reports.ini
    else
        GALAXY_REPORTS_CONFIG=config/reports.ini.sample
    fi
    export GALAXY_REPORTS_CONFIG
fi
```

**Comment by nsoranzo** [comment]
:+1:

**Comment by dannon** [comment]
Thanks!

**Comment by souravsingh** [comment]
Thanks a lot for merging my PR.

**Comment by bgruening** [comment]
Awesome contribution @souravsingh. Keep rocking!

**Pull request merged**